package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "gpt-3-encoder",
-  "version": "1.1.3",
+  "name": "@nick.heiner/gpt-3-encoder",
+  "version": "1.2.0",
```

Comment:
Could we revert the diff on the package.json here?

Reply:
https://github.com/latitudegames/GPT-3-Encoder/pull/30

We can probably do it more incrementally but if you are interested in biting the bullet I made a few more improvements. Passed tests in my fork etc. seem to work well and even got browserify to bundle it all. I did this npm revert to 1.2.0-rc0